config.example.yaml

```diff
@@ -61,50 +61,50 @@ db:
     max_open_connections: 0
     shared: true
     table_prefix: ""
-debug: true
-debug_profiling: true
-disable_itc: true
+debug: true # enable debug mode
+debug_profiling: true # enable endpoints that provides profiling information
+disable_itc: true # enable (ITC) incoming transaction checking
 # Prefixed with "_", because it's unused by default
 graphql:
-  enabled: false
-import_block_headers: ""
+  enabled: false # enables graphql
+import_block_headers: "" # url to import block headers
```

Comment:
Is it still in use?

Reply:
For v1 it is, for v1.1 it will be removed